apps/dotcom/src/app/App.tsx

```diff
@@ -43,6 +44,7 @@ const App: React.FC = () => {
           <Route path="/apply/brand-designer" Component={ApplyBrandDesigner} />
           <Route path="/projects" Component={Projects} />
           <Route path="/people" Component={People} />
+          {/* <Route path="/faq" Component={FAQ} /> */}
```

Comment:
🔧 Is this meant to be commented out?

Reply:
+1 If so, please leave a comment explaining why and when it should be uncommented.